XP-Pen_Artist_10s/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "5543"
+PID: "004a"
+vendor: UC-Logic
+product: XP_Pen_Artist 10.1
```

Comment:
Let's put the same name here and on the directory, use "XP-Pen" as the vendor, and the official model name of "Artist 10S". Let's not put vendor in the model name. I.e. let's have vendor `XP-Pen` and product `Artist 10S`.

The directory should thus be named `XP-Pen_Artist_10S`.

I started writing a README.md trying to outline all the rules I try to follow to preserve consistency here. That should help you nail the contributions faster :)

Reply:
Haha you should give it at beginning :).